### ISSUE TITLE
[Feature Request] TeachingTip

### ISSUE THREAD

**Issue by TheXamlGuy** [Kinnara/ModernWpf#88]
Would be nice to see the TeachingTip ported over from WinUI into this UI Framework.

**Comment by ShankarBUS** [comment]
Hey @Kinnara, I would love to port the **TeachingTip** control from WinUI.

I need this control desperately to use in my app.

I would start my work once #192 is merged because **TeachingTip** requires the `IconSource` classes.

If you don't mind, I would also love to port the new **InfoBar** and **PagerControl** from WinUI.

What do you say?

**Comment by ShankarBUS** [comment]
And how do you generate the xml documentation. I see they are in a separate file instead of being adjacent to the code members.

How do you do it? Do you use any specialized tools?

What about the translation (*.resx) files?
How do you transfer the strings from the *.resw files from WinUI into .NET compatible *.resx files?

All by hand or there any automation tools for this?

**Comment by ShankarBUS** [comment]
Some infos & guidelines on

- How you convert C++/WinRT code from WinUI to C#/WPF?

- How you keep track of the upstream code on the WinUI repo?

- How you manage to keep track of both the XAML Controls Gallery and WinUI's UITests at the same time?

- How the porting should be, a 1:1 port without changing field names and method orders in the classes? Should I also port the helper classes that are required?

- **The Most Important** How do you identify the access modifier of a field/method/class as private/public/internal in C++/WinRT? Should I look at the *.h or *.idl or the *.cpp files?

Would be helpful.

How are you doing it all on your own? Are you even a human 😂?

You've done a great work 👏👏👏.

**Comment by ShankarBUS** [comment]
Nevermind, found a video on YouTube  - https://youtu.be/T3JM0Okda04

This guy ports WinUI SplitButton control to Uno Platform. I'll consider this as a how-to guide.

@Kinnara, I hope you would accept my PR 😅.

**Comment by Kinnara** [comment]
Closing as completed during the 1.x backlog review. The requested behavior is now implemented or superseded by the 1.0.0-preview.1 reboot (#688), or the resolution is already documented in this thread. If a focused problem still reproduces on 1.x, please file a new report against that version.
